.azurePipeline/getPodsLogs.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+"""
+The script will currently be used in the Azure Pipeline to get back the logs of the pods when a test is failing
+allowing developers to document the issue.
+"""
+
+import json
+from pathlib import Path
+import subprocess
+import sys
+
+# The namespace is not configurable as we are expected to run this script only in Azure Dev-Ops which deployed the
+# service only in this namespace.
+NAMESPACE = "test-azure"
+
+
+def get_list_pods(_release_name):
+    cmd = "kubectl --namespace {} get pods -l release={} -o json".format(NAMESPACE, _release_name)
+    r = subprocess.run(cmd.split(" "), capture_output=True)
+    items_from_release = json.loads(r.stdout)['items']
+    return items_from_release
+
+
+def get_logs_container(container_info, file, previous=False):
+    cmd = "kubectl --namespace {} logs {} {}".format(NAMESPACE, container_info['full_pod_name'], container_info['name'])
```

Comment:
Note this command gets logs with ascii color codes... Is this what you want?

Reply:
Use [ansioclors](https://pypi.org/project/ansicolors/) to remove colors from logs.

Reply:
Okay, of course if we are outputting to a console the colors are really good to keep. All good by me either way